api/v2/views/project_volume.py

```diff
@@ -13,7 +13,7 @@ class ProjectVolumeViewSet(AuthViewSet):
     API endpoint that allows instance actions to be viewed or edited.
     """
 
-    queryset = ProjectVolume.objects.all()
+    queryset = ProjectVolume.objects.none()
```

Comment:
Actually, due to https://github.com/iPlantCollaborativeOpenSource/atmosphere/blob/complete_api_v2/api/v2/views/project_volume.py#L20

and the inclusion of 'get_queryset' this whole variable is ignored. It could probably be removed w/o issue.